api/onnx_web/diffusers/run.py

```diff
@@ -241,14 +244,85 @@ def run_inpaint_pipeline(
         mask_filter=mask_filter,
     )
 
+    # check if we can do full-res inpainting if no outpainting is done
+    logger.debug("border zero: %s", border.isZero())
+    if border.isZero():
+        mask_left, mask_top, mask_right, mask_bottom = mask.getbbox()
+        logger.debug("mask bbox: %s", mask.getbbox())
+        mask_width = mask_right - mask_left
+        mask_height = mask_bottom - mask_top
+        # ensure we have some padding around the mask when we do the inpaint (and that the region size is even)
+        adj_mask_size = ceil(max(mask_width, mask_height) * 1.2 / 2) * 2
```

Comment:
Should this padding factor of 1.2 be a parameter? Does it need to increase with very small images, or is 1.2 always good for all sizes?

Reply:
It should probably be a parameter, to be honest. 1.2 worked out to be a good number in testing, but a) that's not a guarantee that that'll be ok for all use cases and b) it certainly wasn't tested with any particular rigor.

In order to implement this, I assume I'll need to change server/params.py to do assignment. Does the ImageParams class also need to change?

Reply:
I have implemented this parameter following the template of strength in img2img; this seems safe to me because this parameter is only used by the inpainting pipeline and because it is not actually needed *inside* the pipeline